app/models/saved_page.rb

```diff
@@ -8,7 +8,9 @@ class SavedPage < ApplicationRecord
   def to_hash
     {
       "page_path" => page_path,
-    }
+      "content_id" => content_id,
```

Comment:
We'll need to update the GDS-API Adaptors and pact tests for these I think?

Reply:
Yeah, I'll update the test helpers and pact tests